admin/commands/inventory/add_agent_postgres_exporter.go

```diff
@@ -88,6 +91,10 @@ func (cmd *AddAgentPostgresExporterCommand) RunCmd() (commands.Result, error) {
 		}
 	}
 
+	if cmd.AutoDiscoveryLimit < 0 {
+		return nil, errors.Errorf("Auto-discovery limit cannot be lower than 0")
+	}
```

Comment:
what about disabling autodiscovery limit at all?

Reply:
@BupycHuk Nailya wanted to show error message in case you will use not valid value. Same check on endpoint side.

Should I change it?

Reply:
do we have the same behaviour for table count?

Reply:
No, negative behaviour for table count was different. Now it is aligned and I added test cases for it.